code/modules/mob/living/carbon/human/human.dm

```diff
@@ -215,6 +215,13 @@
 	if(istype(MB))
 		MB.RunOver(src)
 
+	if(ishuman(AM) && lying)

```

Comment:
Unless you WANT to kill someone by spam stepping on them, it means you don't kill people by stepping on them